tools/pipelines/build-client.yml

```diff
@@ -119,6 +119,9 @@ extends:
     - ci:test:jest
     - ci:test:realsvc:local
     - ci:test:realsvc:tinylicious
+    - ${{ if ne(variables['System.TeamProject'], 'public') }}:
```

Comment:
isn't ci:test:realsvc:local:full a superset of ci:test:realsvc:local? can we avoid redundant work?

Reply:
Yeah, made an improvement